docs/release-notes/v1.0.rst

```diff
@@ -1,6 +1,152 @@
 .. _gammapy_1p0_release:
 
-1.0 (unreleased)
-----------------
+1.0 (October 7th, 2022)
+-----------------------
 
-- No changes yet
+Summary
+~~~~~~~
+
+- Released Oct 7th, 2022
+- 10 contributors
+- 78 pull requests since v0.20.1 (not all listed below)
+
+New features
+~~~~~~~~~~~~
+
+This new release is the Long Term Stable (LTS) version 1.0. Most of the changes are in the package
+infrastructure. A number of improvements and bug corrections have been implemented since v0.20.1.
+A new version of g.a.d.f. has been released (v0.3). It is expected to be compatible with this
+Gammapy version.
```

Comment:
I think "is is expected to..." is too weak and Gammapy is expected to support GADF not the other way around. What about just:

```suggestion
Gammapy v1.0 adds support for latest version v0.3 of `gadf`_.
```